File: service/src/main/java/net/huray/backend/minuting/MinutingApplication.java
package net.huray.backend.minuting;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.data.jpa.repository.config.EnableJpaAuditing;

@SpringBootApplication
@EnableJpaAuditing
public class MinutingApplication {

    public static void main(String[] args) {
        SpringApplication.run(MinutingApplication.class, args);
    }

}

File: service/src/main/kotlin/net/huray/backend/minuting/MinutingApplication.kt
package net.huray.backend.minuting

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.context.properties.ConfigurationPropertiesScan
import org.springframework.boot.runApplication
import org.springframework.data.jpa.repository.config.EnableJpaAuditing

@SpringBootApplication
@EnableJpaAuditing
@ConfigurationPropertiesScan
class MinutingApplication

fun main(args: Array<String>) {
    runApplication<MinutingApplication>(*args)
}
